diff -Nur gcc-4.1.2/libstdc++-v3/acinclude.m4 gcc-4.1.2.patched/libstdc++-v3/acinclude.m4
--- gcc-4.1.2/libstdc++-v3/acinclude.m4	2011-11-11 10:18:50.360994933 -0500
+++ gcc-4.1.2.patched/libstdc++-v3/acinclude.m4	2011-11-11 10:22:53.370994989 -0500
@@ -97,14 +97,14 @@
   # std::exit can be guessed by AC_PROG_CXX, and used in later tests.
 
   m4_define([ac_cv_prog_CXX],[glibcxx_cv_prog_CXX])
-  m4_rename([_AC_ARG_VAR_PRECIOUS],[glibcxx_PRECIOUS])
+  m4_rename_force([_AC_ARG_VAR_PRECIOUS],[glibcxx_PRECIOUS])
   m4_define([_AC_ARG_VAR_PRECIOUS],[])
   save_CXXFLAGS="$CXXFLAGS"
   CXXFLAGS="$CXXFLAGS -fno-builtin"
   AC_PROG_CC
   AC_PROG_CXX
   CXXFLAGS="$save_CXXFLAGS"
-  m4_rename([glibcxx_PRECIOUS],[_AC_ARG_VAR_PRECIOUS])
+  m4_rename_force([glibcxx_PRECIOUS],[_AC_ARG_VAR_PRECIOUS])
   AC_SUBST(CFLAGS)
   AC_SUBST(CXXFLAGS)
 
